internal/cephfs/volume.go

```diff
@@ -85,9 +83,21 @@ func getVolumeRootPathCeph(ctx context.Context, volOptions *volumeOptions, cr *u
 	return strings.TrimSuffix(string(stdout), "\n"), nil
 }
 
+type additionalClusterInfo struct {
+	resizeSupported       bool
+	subvolumegroupCreated bool
+}
+
 func createVolume(ctx context.Context, volOptions *volumeOptions, cr *util.Credentials, volID volumeID, bytesQuota int64) error {
-	//TODO: When we support multiple fs, need to hande subvolume group create for all fs's
-	if !cephfsInit {
+	// keyPresent checks whether corresponding clusterID key is present in clusterAdditionalInfo
+	var keyPresent bool
```

Comment:
you don't need keyPresent as a separate variable.

Reply:
Currently keypresent variable mainly helps in initializing the struct with default values which might otherwise throw an error. Since keypresent is also used outside it's scope, the variable was created. Should I find a workaround?

Reply:
where it is used outside of loop in createVolume?

Reply:
Here https://github.com/ceph/ceph-csi/pull/1175/files#diff-fa3ab9f6b6a9e75fdc1e69a6ee650d9eR167 ; to try resize for the first run instead of create.